tools/packrom/packrom.cpp

```diff
@@ -67,9 +73,10 @@ typedef struct{
 	u8 author[32];
 	u8 icon[16*16];
 	u32 crc32;
-	u8 mouse;
+	u8 psupport; //peripherals supported
```

Comment:
I'm assuming this is backwards compatible because the only valid value for the previous "mouse"  field is 0 or 1, correct?